vendor/double-conversion/upstream/double-conversion/bignum-dtoa.cc

```diff
@@ -370,7 +370,7 @@ static void BignumToFixed(int requested_digits, int* decimal_point,
 // Returns an estimation of k such that 10^(k-1) <= v < 10^k where
 // v = f * 2^exponent and 2^52 <= f < 2^53.
 // v is hence a normalized double with the given exponent. The output is an
-// approximation for the exponent of the decimal approimation .digits * 10^k.
```

Comment:
Similar to the above comment, this comes from the upstream Double-Conversion repo, so ideally it should be fixed upstream.